Makefile

```diff
@@ -9,10 +9,8 @@ build:
 check-gofmt:
 	scripts/check_gofmt.sh
 
-# This command is overcomplicated because Golint's `./...` doesn't filter
-# `vendor/` (unlike every other Go command).
 lint:
-	go list ./... | xargs -I{} -n1 sh -c 'golint -set_exit_status {} || exit 255'
+	golint
```

Comment:
golint is deprecated, and there's not much we can do to customize it's behavior. We have another task to replace it with a supported linter.

Reply:
https://github.com/stripe/stripe-go/pull/1288 is what we did for stripe-go

Reply:
Updated to be the same in this repo.